src/components/NetworkSelector/NetworkSelector.js

```diff
@@ -48,10 +52,11 @@ function NetworkSelector(props) {
   return (
     <div>
       <Dropdown isOpen={dropdownOpen} toggle={toggle}>
-        <DropdownToggle caret tag="span" className={styles.dropdownToggle}>
```

Comment:
why is this changed?

Reply:
If you type in url  /v3/arbitrum and it calls currentNetwork it crashes if arbitrum is not in the network selector